chrome/content/zotero/components/form/input.jsx

```diff
@@ -68,8 +68,8 @@ class Input extends React.PureComponent {
 		}
 	}
 
-	handleChange({ target }, options) {
-		var newValue = options.newValue || target.value;
+	handleChange({ target }, { newValue } = {}) {
+		var newValue = newValue || target.value;
```

Comment:
No `var` here — the variable is already declared by its use as an argument.

Reply:
Oh, sure! I was using the online editor and missed that.